src/app/utils/persistentStorageTemplate.ts

```diff
@@ -0,0 +1,54 @@
+export const fileStore = `"file-store": {
+  "shared": false,
+  "data": {
+    "path": "data"
+  },
+  "index": {
+    "path": "index"
+  },
+  "write-behind": {
+    "modification-queue-size": "2048"
+  }
+}`;
+
+export const remoteStore = `"remote-store": {
+  "cache": "mycache",
+  "raw-values": "true",
+  "remote-server": [
+    {
+      "host": "one",
+      "port": "12111"
+    },
+    {
+      "host": "two"
+    }
+  ],
+  "connection-pool": {
```

Comment:
We can remove this object and let the user use the default connection-pool configuration.